docs/docs/examples/vector_stores/AzureCosmosDBNoSqlDemo.ipynb

```diff
@@ -0,0 +1,465 @@
+    "    model=\"AZURE_OPENAI_EMBEDDING_MODEL\",\n",
+    "    deployment_name=\"AZURE_OPENAI_EMBEDDING_MODEL_DEPLOYMENT_NAME\",\n",
+    "    azure_endpoint=\"AZURE_OPENAI_BASE\",\n",
+    "    api_key=\"AZURE_OPENAI_KEY\",\n",
+    "    api_version=\"AZURE_OPENAI_VERSION\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "8da35d45-9689-4f3a-9011-1cda0fb361ea",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from llama_index.core import Settings\n",
+    "\n",
+    "Settings.llm=llm\n",
+    "Settings.embed_model=embed_model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "084aa964-7222-47b2-bdab-825c85a6ffed",
+   "metadata": {},
+   "source": [
+    "# Loading Documents\n",
+    "\n",
+    "In this example we will be using the paul_graham essay which will be processed by the SimpleDirectoryReader."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "8f689978-93c6-4c34-9a6e-9fca606a1058",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Document ID: 02615dc0-7189-4b4a-ad6b-f6ebc44f1364\n"
+     ]
+    }
+   ],
+   "source": [
+    "from llama_index.core import SimpleDirectoryReader\n",
+    "\n",
+    "documents = SimpleDirectoryReader(input_files=[r\"\\examples\\data\\paul_graham\\paul_graham_essay.txt\"]).load_data()\n",
+    "\n",
+    "print(\"Document ID:\", documents[0].doc_id)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e6c5f4bf-411e-482d-8ada-580dad6575ee",
+   "metadata": {},
+   "source": [
+    "# Create the index\n",
+    "\n",
+    "Here we establish the connection to cosmos db nosql and create a vector store index."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "4eb1251c-8bbb-416d-9c32-c7260d039900",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.cosmos import CosmosClient, PartitionKey\n",
+    "from llama_index.vector_stores.azurecosmosnosql import AzureCosmosDBNoSqlVectorSearch\n",
+    "from llama_index.core import StorageContext\n",
+    "\n",
+    "#create cosmos client\n",
+    "URL = 'AZURE_COSMOS_CONNECTION_STRING'\n",
+    "KEY = 'AZURE_COSMOS_KEY'\n",
+    "client = CosmosClient(URL, credential=KEY)\n",
+    "\n",
+    "#specify vector store properties\n",
+    "indexing_policy = {\n",
+    "    \"indexingMode\": \"consistent\",\n",
+    "    \"includedPaths\": [{\"path\": \"/*\"}],\n",
+    "    \"excludedPaths\": [{\"path\": '/\"_etag\"/?'}],\n",
+    "    \"vectorIndexes\": [{\"path\": \"/embedding\", \"type\": \"quantizedFlat\"}],\n",
+    "}\n",
+    "\n",
+    "vector_embedding_policy = {\n",
+    "    \"vectorEmbeddings\": [\n",
+    "        {\n",
+    "            \"path\": \"/embedding\",\n",
+    "            \"dataType\": \"float32\",\n",
+    "            \"distanceFunction\": \"cosine\",\n",
+    "            \"dimensions\": 3072,\n",
```

Comment:
I know you told me the sample works, but is there a reason for the dimensions to be 3072?

Reply:
I tested out the dimensions at 3072 to see if the answer changes when you increase the dimension size. A smaller dimension size works fine too.